deeplake/core/vectorstore/test_deeplake_vectorstore.py

```diff
@@ -262,53 +264,69 @@ def filter_fn(x):
     assert isinstance(data_p_v.text[0].data()["value"], str)
     assert data_p_v.embedding[0].numpy().size > 0
 
+    # Check that specifying exec option during search works
+    _ = vector_store.search(
+        embedding=query_embedding,
+        exec_option="python",
```

Comment:
shouldn't you just change this to "tensor_db"?

Reply:
Yeah, not sure why this says "python". I thought I made the change. Maybe I accidentally messed something up. Will change it back.

Reply:
@adolkhan can you check now pls.